cpp/src/arrow/flight/CMakeLists.txt

```diff
@@ -19,12 +19,8 @@ add_custom_target(arrow_flight)
 
 arrow_install_all_headers("arrow/flight")
 
-set(ARROW_FLIGHT_STATIC_LINK_LIBS
```

Comment:
This file needs to be cmake-formatted.

Reply:
https://arrow.apache.org/docs/developers/cpp/development.html#code-style-linting-and-ci

Reply:
hey @lidavidm i did run the cmake formatter. this is the output. This is what led me to start playing with `.pre-commit-config.yaml`. I am using cmake-format `0.6.9` if that makes a difference. I have reverted this and will address formatting on the mailing list